addons/survey/views/survey_templates.xml

```diff
@@ -21,12 +21,25 @@
             <div id="wrap" class="oe_structure oe_empty"/>
         </xpath>
         <xpath expr="//footer" position="after">
-            <div class="py-3">
-                <div class="float-right px-5 py-2 bg-white rounded-left">
-                    <t t-call="web.brand_promotion_message">
+            <div class="py-3 m-0 p-0 text-right">
+                <div class="o_survey_progress_wrapper d-inline-block pr-5 text-left">
+                    <t t-call="survey.survey_progression"
```

Comment:
Why are you calling survey_progression that is also rendered by a route and added by JS ? Is it for some initialization ?

Reply:
It is rendered by the JS side only when you submit, because the page is not fully refreshed (AJAX magic + fade out / fade in of next question -> we need to update the progression).

But it also needs to work when you fully refresh the page, then the template is rendered and the JS does not handle anything (exact same mechanic as the form content).